extensions/git/src/util.ts

```diff
@@ -149,6 +149,16 @@ export async function mkdirp(path: string, mode?: number): Promise<boolean> {
 	return true;
 }
 
+export async function createOrAppendFile(path: string, data: string, options?: any): Promise<boolean> {
```

Comment:
Yes, any change to the model is changed automatically if auto save is enabled. We open the file automatically in the background if a model becomes dirty and was not opened yet.